joins.sql

```diff
@@ -0,0 +1,71 @@
+-- SELECT *
+-- FROM users;
+
+-- SELECT *
+-- FROM users
+-- WHERE userID = 100;
+
+-- SELECT posts.*, users.first_name, users.last_name
+-- FROM posts
+--     INNER JOIN users on posts.userID = users.userID
+-- WHERE posts.userID = 200;
+
+-- SELECT posts.*, users.username
+-- FROM posts
+--     INNER JOIN users ON posts.userID = users.userID
+-- WHERE users.first_name = 'Norene' AND users.last_name = 'Schmitt';
+
+-- SELECT username
+-- FROM users
+-- WHERE created_at > '2015-01-01';
+
+-- SELECT posts.title, posts.content, users.username
+-- FROM posts
+--     INNER JOIN users on posts.userID = users.userID
+-- WHERE users.created_at < '2015-01-01';
+
+-- SELECT comments.*, posts.title AS "Post Title"
+-- FROM comments
+--     INNER JOIN posts on comments.postID = posts.postID;
+
+-- SELECT posts.title AS post_title, posts.url AS post_url, comments.body AS comment_body
+-- FROM comments
+--     INNER JOIN posts on comments.postID = posts.postID
+-- WHERE posts.created_at < '2015-01-01';
+
+-- SELECT posts.title AS post_title, posts.url AS post_url, comments.body AS comment_body
+-- FROM comments
+--     INNER JOIN posts on comments.postID = posts.postID
+-- WHERE posts.created_at > '2015-01-01';
+
+-- SELECT posts.title AS post_title, posts.url as post_url, comments.body AS comment_body
+-- FROM comments
+--     INNER JOIN posts on comments.postID = posts.postID
+-- WHERE comments.body LIKE '%USB%';
+
+-- SELECT posts.title AS post_title, users.first_name, users.last_name, comments.body AS comment_body
+-- FROM comments
+--     INNER JOIN users on users.userID = comments.userID
```

Comment:
User should be referenced by the post's user_id. Right now you're grabbing the comment's user and returning their first and last name.